tests/test_scalars.py

```diff
@@ -1530,6 +1531,8 @@ def test_timestamp_misc_methods() -> None:
         pd.Timestamp,
     )
 
+    check(assert_type(ts2.as_unit('ns'), pd.Timestamp), pd.Timestamp)
```

Comment:
```suggestion
    check(assert_type(ts2.as_unit("ns"), pd.Timestamp), pd.Timestamp)
```
use the pre-commit hook.  You need double quotes here.